app/bundles/LeadBundle/Translations/en_US/messages.ini

```
@@ -913,6 +913,14 @@ mautic.widget.segments.build.time="Segments build time"
 mautic.widget.segments.build.time.longest="Longest time"
 mautic.widget.segments.build.time.shortest="Shortest time"
 mautic.widget.created.leads="Created contacts"
+mautic.widget.number.contacts="Number of contacts"
```

As per the code it's excluding the anonymous contacts I'd call it
```suggestion
mautic.widget.number.contacts="Number of identified contacts"
```
```suggestion
mautic.widget.number.contacts="Number of contacts"
```
```suggestion
mautic.widget.number.contacts="Number of contacts"
```